po/fr.po

```diff
@@ -157,11 +178,11 @@ msgstr "$%lc est un nom de variable invalide dans fish."
 
 #, c-format
 msgid "$%ls: originally inherited as |%ls|\n"
-msgstr ""
+msgstr "$%ls : hérité comme |%ls|\n"
 
 #, c-format
 msgid "$%ls: set in %ls scope, %ls,%ls with %d elements"
-msgstr ""
+msgstr "$%ls : paramétré dans la portée de %ls, %ls,%ls avec %d éléments"
```

Comment:
For German translations I quoted the scope name parameter (like 'local', using the wrong quotes though).
Not sure if that is better or worse.

```suggestion
msgstr "$%ls : paramétré dans la portée « %ls », %ls,%ls avec %d éléments"
```